src/sctools/test/test_metrics.py

```diff
@@ -33,10 +33,12 @@
 # set the input files
 _gene_sorted_bam = _data_dir + '/small-gene-sorted.bam'
 _cell_sorted_bam = _data_dir + '/small-cell-sorted.bam'
+_cell_sorted_bam_missing_cell_barcodes = _data_dir + '/cell-sorted-missing-cb.bam'
```

Comment:
would use `os.path.join()` instead of adding the slash for a bit more generality. Also line 41 below.